src/dashboard/alerts/alerts.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
Is this file needed? It seems like it is just an example file. We don't want to install unnecessary files on the router.

Reply:
Not necessary, thought it would be good to have examples in the repo but I can remove it.